org.envirocar.app/src/org/envirocar/app/views/tracklist/TrackListLocalCardFragment.java

```diff
@@ -543,4 +564,10 @@ private void updateProgressView(int progress) {
             progressText.setText(progress + " / " + numTracks);
         }
     }
+    private boolean isNetworkAvailable() {
+        ConnectivityManager connectivityManager
+                = (ConnectivityManager) getActivity().getSystemService(Context.CONNECTIVITY_SERVICE);
+        NetworkInfo activeNetworkInfo = connectivityManager.getActiveNetworkInfo();
+        return activeNetworkInfo != null && activeNetworkInfo.isConnected();
+    }
```

Comment:
There is already a helper class for checking the internet connection: https://github.com/enviroCar/enviroCar-app/blob/master/org.envirocar.core/src/main/java/org/envirocar/core/ContextInternetAccessProvider.java

Please, reuse it instead of reimplementing it.